src/app/features/home/search-bar/store/search-bar.reducer.ts

```diff
@@ -0,0 +1,14 @@
+import { createReducer, on } from '@ngrx/store';
+import { SearchBarActions } from './search-bar.actions';
+import { initialSearchBarState } from './search-bar.state';
+
+export const searchBarReducer = createReducer(
+  initialSearchBarState,
+  on(SearchBarActions.upadate_search_results, (state, { events, groups }) => {
+    return {
+      ...state,
+      events: events,
+      groups: groups,
+    };
+  })
+);
```

Comment:
zastanawiam się nad sensem trzymania eventów oraz grup w storze? Czemu ma to nam służyć?